src/client.ts

```diff
@@ -265,6 +293,58 @@ export class AutoblocksAPIClient {
     );
   }
 
+  public async getLocalRuns(testExternalId: string): Promise<{
```

Comment:
let's prefix all the methods here with Test so

getLocalTestRuns

getCITestRuns

GetLocalTestResults

and so on...